tests/fixtures/synthetic_hf_state.py

```diff
@@ -14,15 +14,16 @@
 from tests.datasets import test_synthetic_lm_data
 
 
-def make_dataset_configs() -> list:
+def make_dataset_configs(model_family=('bert', 'gpt2')) -> list:
     lm_dataset_configs = [
         config[0] for config in test_synthetic_lm_data.generate_parameter_configs(
-            ['num_samples', 'chars_per_sample', 'column_names', 'tokenizer_family'])
+            ['num_samples', 'chars_per_sample', 'column_names', 'tokenizer_family'], model_family=model_family)
     ]
     return lm_dataset_configs
 
 
 def make_lm_tokenizer(config: dict):
+    pytest.importorskip('transformers')
```

Comment:
skipping here could be a cause as well of `tests/test_synthetic_hf_state.py` being skipped.